uiowa/uiowa
# Fix menu redirect to direct users to current menu instead of menu list

Closes #4310 

P.S. Is it really going rogue when you do it during your free time? 😇
# How to test


Steps to reproduce the issue:

`composer install`
`blt ds --site=default`
Go to admin/structure/menu
Click on any edit menu
Click on add link/or edit existing link
Fill out the add link form and press save
A user would expect to return to the menu with its list of links. (instead of current regression where you return to the list of menus)
